ci/ray_ci/pipeline/gap_filling_scheduler.py

```diff
@@ -55,8 +57,19 @@ def get_gap_commits(self) -> List[str]:
         )
 
     def _trigger_build(self, commit: str) -> int:
-        # TODO(can): Implement this method
-        pass
+        build = [
+            build
+            for build in self._get_builds()
+            if build["commit"] == commit and build["state"] == "blocked"
+        ][0]
+        self.buildkite.jobs().unblock_job(
+            self.buildkite_organization,
+            self.buildkite_pipeline,
+            build["number"],
+            build["jobs"][0]["id"],  # first job is the blocked job
```

Comment:
is it possible to check some thing that is more reliable? like the label or the step?